api/server/router/build/build_routes.go

```diff
@@ -92,6 +50,7 @@ func newImageBuildOptions(ctx context.Context, r *http.Request) (*types.ImageBui
 	options.CPUSetCPUs = r.FormValue("cpusetcpus")
 	options.CPUSetMems = r.FormValue("cpusetmems")
 	options.CgroupParent = r.FormValue("cgroupparent")
+	options.Tags = r.Form["t"]
```

Comment:
@jhowardmsft : Trying to get context of your comment. Does it mean that currently the Windows CI has a subset of builder tests and that the complete set is a WIP?